.gitignore

```diff
@@ -191,6 +191,7 @@ node_modules/
 /ajax/libs/jquery-easing/1.3.0/
 /ajax/libs/jquery-easing/1.3.1/
 /ajax/libs/libphonenumber/libphonenumber*/
+/ajax/libs/slippry/v.1.2/
```

Comment:
@maruilian11 Is not v.1.2?
![ss](https://cloud.githubusercontent.com/assets/13430892/23825610/2611edfe-068d-11e7-88b7-3734e3d5958a.png)